src/ert/config/gen_kw_config.py

```diff
@@ -120,10 +117,15 @@ def from_config_list(cls, gen_kw: list[str | dict[str, str]]) -> Self:
 
         options = cast(dict[str, str], gen_kw[-1])
         positional_args = cast(list[str], gen_kw[:-1])
-        forward_init = str_to_bool(options.get("FORWARD_INIT", "FALSE"))
-        init_file = _get_abs_path(options.get("INIT_FILES"))
-        update_parameter = str_to_bool(options.get("UPDATE", "TRUE"))
         errors = []
+        update_parameter = str_to_bool(options.get("UPDATE", "TRUE"))
+        if _get_abs_path(options.get("INIT_FILES")):
+            raise ConfigValidationError(
```

Comment:
Is it possible to add some context, pointing to the correct line number in the config file?

Reply:
Fixed!